exopy_hqc_legacy/instruments/drivers/visa/agilent_pna.py

```diff
@@ -177,11 +176,9 @@ def run_averaging(self, aver_count=''):
                     done = self._pna.ask_for_values('*OPC?')[0]
                     break
                 except Exception:
-                    self._pna.timeout = self._pna.timeout*2
```

Comment:
I do not really like the idea that this can deadlock forever even if for example the PNA shutdown. I would prefer to catch a more specific error (VisaIOError) and check that we did get an actual timeout instead of any other error.

Reply:
Good idea, I'm going to run a few tests to make sure I only catch the correct error type.

Reply:
I am now only catching VisaIOError that are caused by a timeout. This means that an instrument disconnect will raise an exception now instead of staying in this infinite loop.